### DISCUSSION TITLE
Is githubclone.com legit or a phishing site?

### DISCUSSION THREAD

**Discussion by data-miner00** [community/community#137304] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

![image](https://github.com/user-attachments/assets/984ccb1c-ad8f-489a-8b81-aa7e7053b803)

Something has been bothering me about this suspicious [githubclone.com](https://www.githubclone.com/) that I've randomly discovered while browsing the web months ago. It has the exact same landing page as GitHub here and most of it's navigation links will just point to the legit GitHub (github.com) domain. However, the sign in/sign up page is still at githubclone.com. This website is not Google-searchable.

A cursory lookup on [whois.com](https://www.whois.com/whois/githubclone.com) shows that it is not purchased by GitHub. This is what the official [GitHub whois result](https://www.whois.com/whois/github.com) looks like. I tried to create a ticket for this but couldn't find any helpful links in the docs(I might have missed it). Appreciate it if anyone could chip in for my query, thanks!

**Comment by davevad93** [comment]
Hi @data-miner00 ,

I've seen discussions in the past about fake github sites, I don't know if it was this one in particular, but personally I think it's 100% a scam site. 

There's no section on the GitHub support page for this sort of things, but if you want you can open a ticket [here](https://support.github.com/contact/account), select `Other` and ask them, although I think opening a discussion here in the Community as you've done is a valid alternative.

**Reply by data-miner00** [reply]
Hi @davevad93 ,

Thanks for the swift reply and the link to create the ticket. I'll get back to here once I hear back from the GitHub team. Cheers!

**Reply by davevad93** [reply]
You're welcome @data-miner00 👍

**Comment by ebndev** [comment, marked as answer]
Hi @data-miner00, 👋🏻 thanks for flagging this. I've gone ahead and reported this to the proper GitHub team.

**Reply by data-miner00** [reply]
Hi @ebndev , thanks for the reply, I think at this point we can confirm that it is an ill-intended site, thanks!

**Comment by daiplusplus** [comment]
@ebndev  1.5 years later (February 2026), and it seems `githubclone.com` is back...

<img width="2601" height="1857" alt="image" src="https://github.com/user-attachments/assets/83040641-0f35-41d5-83d3-049f0d48a126" />

Concerningly, their cloned login-page is still ripe for harvesting credentials - I think they just copied GH's site identically right down to the CSP and CloudFlare scripts, which I guess explains why CloudFlare gives me a warning after I posted fake login details:

<img width="2791" height="2238" alt="image" src="https://github.com/user-attachments/assets/f6f9705b-4ebf-4ff5-88b1-2184c164a30c" />

--------

T[he WHOIS for the domain still points to `markmonitor.com`](https://lookup.icann.org/en/lookup) (who I assume are GitHub's official people), so I guess the registration lapsed and markmonitor failed to renew it, so then the _ne'er-do-wells_ snapped it up and didn't even bother to update the whois.
